packages/legend-studio/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_MappingTransformer.ts

```diff
@@ -583,10 +594,14 @@ const transformOtherwiseEmbeddedRelationalPropertyMapping = (
 
 const transformXStorePropertyMapping = (
   element: XStorePropertyMapping,
+  isTransformingEmbeddedPropertyMapping: boolean,
```

Comment:
I don't recall we have a bug with `XStore` @MauricioUyaguari ?

Reply:
nope there was no bug. this was to make it consistent with other methods.

Reply:
see my comment below